decidim-meetings/app/services/decidim/meetings/calendar/component_calendar.rb

```diff
@@ -12,7 +12,7 @@ class ComponentCalendar < BaseCalendar
         # Returns a String.
         def events
           Rails.cache.fetch(cache_key) do
-            meetings.map do |meeting|
+            filtered_meetings.map do |meeting|
```

Comment:
I believe this would introduce the same problem that was fixed at #8988.

The active filters get into the cache and the next time they are requested, the cache entry contains the active filters at the time of caching.

Either we need to take the filters into account in the `cache_hash` calculation or remove caching from this part.